openwisp_firmware_upgrader/templates/admin/firmware_upgrader/batch_upgrade_operation_change_form.html

```diff
@@ -0,0 +1,160 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls static admin_list ow_tags %}
+
+{% block extrahead %}
+{{ block.super }}
+<link rel="stylesheet" type="text/css" href="{% static "admin/css/changelists.css" %}">
+<link rel="stylesheet" type="text/css" href="{% static "admin/css/ow-filters.css" %}">
+<link rel="stylesheet" type="text/css" href="{% static "firmware-upgrader/css/batch-upgrade-operation.css" %}">
+<script src="{% static "admin/js/ow-filter.js" %}"></script>
+<script type="text/javascript" src="{% static 'admin/js/jquery.init.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<!-- Upgrade Operations Section -->
+<h1 class="upgrade-operations-title">{% trans "Upgrade Operations" %}</h1>
+        
+        <!-- OpenWISP Filter Section -->
```

Comment:
the indentation is odd here. Please make sure to use 2 spaces indentation on HTML templates.